src/R/Support/Impl/Help/Packages/PackageInfo.cs

```diff
@@ -77,10 +77,8 @@ internal sealed class PackageInfo : NamedItemInfo, IPackageInfo {
         private async Task<IEnumerable<string>> GetFunctionNamesAsync() {
             var functions = TryRestoreFromCache();
             if (functions == null || !functions.Any()) {
-                try {
-                    var result = await _host.Session.PackagesFunctionsNamesAsync(Name, REvaluationKind.BaseEnv);
-                    functions = result.Children<JValue>().Select(v => (string)v.Value);
-                } catch (TaskCanceledException) { } catch (REvaluationException) { }
+                var result = await _host.Session.PackagesFunctionsNamesAsync(Name, REvaluationKind.BaseEnv);
+                functions = result.Children<JValue>().Select(v => (string)v.Value);
```

Comment:
Previous implementation allowed to proceed with next package in case of error. With this change, `LoadRemainingPackagesFunctions` will stop on the first error. Is it intentional?